### PR TITLE
working-groups: add test infra working group

### PR DESCRIPTION
**Pull request by you06**
Signed-off-by: you06 <you1474600@gmail.com>

Add "Test Infra" working group.

### CODE REVIEW

File: working-groups/wg-test-infra.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Test Infra Working Group
```
Comment (zz-jason) [inline working-groups/wg-test-infra.md]: please update the `working-groups/README.md` file.

**Comment by you06** [comment]
@zz-jason PTAL again.

File: working-groups/wg-test-infra.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* [5kbpers](https://github.com/5kbpers), [PingCAP](https://github.com/pingcap)
+* [amyangfei](https://github.com/amyangfei), [PingCAP](https://github.com/pingcap)
+* [cwen0](https://github.com/cwen0), [PingCAP](https://github.com/pingcap)
+* [mahjonp](https://github.com/mahjonp), [PingCAP](https://github.com/pingcap)
+* [YangKeao](https://github.com/YangKeao), [PingCAP](https://github.com/pingcap)
+* [you06](https://github.com/you06), [PingCAP](https://github.com/pingcap)
+
+## Contact
+
+* Slack: channel **#wg-test-infra** in the
```
Comment (gingerkidney) [inline working-groups/wg-test-infra.md]: Please build the slack channel

Comment (you06) [reply working-groups/wg-test-infra.md]: Done

**Pull request merged**